AirCasting/Utils/Connection/BluetoothConnection.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+import CoreData
+import Resolver
+
+final class BluetoothConnectionProtector: ConnectionProtectable {
+    
+    enum BluetoothConnectionProtectorError: Error {
+        case alreadyConnected
+        case readError(Error)
+    }
+    
+    @Injected private var database: SessionsFetchable
+    
+    func isAirBeamAvailableForNewConnection(completion: @escaping (Result<Void, Error>) -> Void) {
+        let predicate = NSPredicate(format: "deviceType == %@ AND type == %@ AND (status == %li || status == %li || status == %li)",
+                                    "1",
```

Comment:
don't we have something like AirBeam.rawValue?

Reply:
Yes we do, IDK why it does not work but it does not and I was trying hard 😿